lib/slackabot.ex

```diff
@@ -2,6 +2,7 @@ defmodule Slackabot do
   alias Slackabot.Slack
 
   def start(_type, _args) do
+    Slackabot.Repo.start_link
```

Comment:
I think you might be able to get mixfile to manage this ? as part of the application